packages/autocomplete-core/src/onInput.ts

```diff
@@ -43,10 +38,6 @@ export function onInput<TItem extends BaseItem>({
   store,
   ...setters
 }: OnInputParams<TItem>): Promise<void> {
-  nextStateRef = nextState;
-  propsRef = props;
-  queryRef = query;
```

Comment:
Why delete those? When you're not in a `shouldSkipSearch` situation, you'd need those because you're not guaranteed to have the last called `onInput` resolve last.

Reply:
I relied on the tests for the implementation and this doesn't seem to be covered then. We can add it back and test it.

Reply:
Yes let's test it in the base PR.